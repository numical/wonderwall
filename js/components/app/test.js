'use strict';
/* eslint-env jasmine */

import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import App from './index.js';

describe('App ', () => {
  it('should match previous snapshot', () => {
    const app = shallow(<App />);
    expect(shallowToJson(app)).toMatchSnapshot();
  });

  it('should say Hello world!', () => {
    const app = shallow(<App />);
    expect(app.find('div').text()).toEqual('Hello World!');
  });
});

