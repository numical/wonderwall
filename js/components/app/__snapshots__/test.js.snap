exports[`App  should match previous snapshot 1`] = `
<div>
  Hello World!
</div>
`;
